src/Benchmarks.ServerJob/ServerCounter.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Benchmarks.ServerJob
+{
+    public class ServerCounter
+    {
+        public long WorkingSet { get; set; }
+        public double CpuPercentage { get; set; }
```

Comment:
I would add a `TimeSpan Elapsed` property to indicate how soon after process start the metric was obtained.